### PR TITLE
Update action_list.rst

### PR DESCRIPTION
**Pull request by mrdavecox**
Problem: 
The documentation currently lacks instructions regarding how to configure the default ordering in the list view using multiple columns.

Solution: 
I have made changes to the documentation showing how to override the createQuery method (based on code kindly provided here: https://github.com/sonata-project/SonataAdminBundle/issues/1548). However, one of the problems with overriding the createQuery method completely is it then stops admin users being able to sort columns by clicking on the column headings. The proposed changes check the filter parameter of the query string to see whether the user is viewing the default view or whether a filter has been applied. If the filter has been applied, then the parent method is returned as normal, otherwise the custom query is created and returned by the createQuery class.

The example uses a simple Product/Category entity relationship, as this would seem a pretty common case for when developers would want to sort by multiple columns when users arrive at the default Product list view.

### CODE REVIEW

File: Resources/doc/reference/action_list.rst [hunk 1]
```diff
@@ -213,6 +213,45 @@ the ``datagridValues`` array property. All three keys ``_page``, ``_sort_order``
...
+        
+        if( $filter ) {
+            // If the filter has been set, filter the results as normal
+            $query = parent::createQuery($context);       
+        }
+        else {
+            // The filter has not been set, so create an initial query for the list view
+            // using multiple columns
+            $qb = $this->getModelManager()->getEntityManager($this->getClass())->createQueryBuilder();
+            $qb->select( array( 'p', 'c' ))
```
Comment (Soullivaneuh) [inline Resources/doc/reference/action_list.rst]: Please respect Symfony CS.

File: Resources/doc/reference/action_list.rst [hunk 2]
```diff
@@ -213,6 +213,45 @@ the ``datagridValues`` array property. All three keys ``_page``, ``_sort_order``
...
+                                 ->from($this->getClass(), 'p')
```
Comment (Soullivaneuh) [inline Resources/doc/reference/action_list.rst]: Less indentation please.

**Comment by Soullivaneuh** [comment]
:+1:

**Comment by mrdavecox** [comment]
I'd probably just not bother with this update now. Like I said in my previous comment, as soon as you navigate to another page the ordering is forgotten anyway, so it is of little use as SonataAdmin doesn't support multi column ordering...

**Pull request closed without merging**